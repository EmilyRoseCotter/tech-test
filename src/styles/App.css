.App {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.logo {
    width: 17em;
}