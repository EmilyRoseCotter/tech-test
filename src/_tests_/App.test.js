import React from "react";
import { render, screen } from '@testing-library/react';
import App from '../components/App';

describe("App", () => {
  test("redners App component correctly", () => {
    const { asFragment } = render(<App/>);

    expect(asFragment()).toMatchSnapshot(); 
  })
})
