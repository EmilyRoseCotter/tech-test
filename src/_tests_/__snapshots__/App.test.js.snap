// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App redners App component correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="NasaLogo"
      class="logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <div
      class="Search"
    >
      <form
        class="search-form"
      >
        <input
          class="search-input"
          type="text"
        />
        <button
          class="search-button"
          type="submit"
        >
          Go!
        </button>
      </form>
    </div>
    <p
      class="nr-title"
    >
      Please enter a valid search term
    </p>
  </div>
</DocumentFragment>
`;
